dist/obs-bundled-gems.spec

```
@@ -16,7 +16,8 @@
 #
 
 %if 0%{?suse_version}
-%define __obs_ruby_interpreter /usr/bin/ruby.ruby3.1
+%define __obs_ruby_suffix      ruby3.1
+%define __obs_ruby_interpreter /usr/bin/ruby.%{__obs_ruby_suffix}
```

```suggestion
%define __obs_ruby_suffix      .ruby3.1
%define __obs_ruby_interpreter /usr/bin/ruby%{__obs_ruby_suffix}
```

Could it be possible to use the same suffix for both `__obs_ruby_suffix` and `OBS_RUBY_SUFFIX`, starting both with a dot (`.`)?

---

the current solution has the advantage that if you do not want to override the setting you could use

```
%define __obs_ruby_suffix %rb_default_ruby_suffix
```

though i can change the code in the make file to use the dot-less version.